libsmtutil/CVC5Interface.cpp

```diff
@@ -0,0 +1,319 @@
+/*
```

Comment:
Unfortunately, the diff seems to be too large for `git` to treat it only as a rename. Should I try to separate the renaming and the actual changes in the file?

Reply:
I have separated the renaming of `CVC4Interface` files, so now if you look at the changes from only first commit, you can see the changes in those file side-by-side.